Cargo.toml

```diff
@@ -179,6 +179,7 @@ semver = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 serde_yaml = { workspace = true }
+shellexpand = "3.1.0"
```

Comment:
would you mind using `{workspace = true }` here and moving this line to workspace dependencies?